Description: Fix buggy POD section in IniFiles.pm, which causes
 pod2man to create buggy manpage for IniFiles. There is a =cut
 closing an unexistent POD block.
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>
Status of the patch: First integrated in libconfig-inifiles-perl 
 2.49-1. Patch sent upstream on 2009-05-31
--- a/lib/Config/IniFiles.pm
+++ b/lib/Config/IniFiles.pm
@@ -2461,10 +2461,6 @@
 
 1;
 
-=cut
-
-1;
-
 # Please keep the following within the last four lines of the file
 #[JW for editor]:mode=perl:tabSize=8:indentSize=2:noTabs=true:indentOnEnter=true:
 
